src/stores/BreadcrumbsStore.ts

```diff
@@ -51,7 +52,10 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
     }
 
     public get visible(): boolean {
-        return this.state.enabled;
+        // @ts-ignore - TypeScript really wants this to be [tagId: string] but we know better.
+        const roomCount = _reduce(RoomListStoreTempProxy.getRoomLists(), (result, rooms) => result + rooms.length, 0)
+        console.log(`calculating roomlist size: ${roomCount}`)
+        return roomCount >= 20;
```

Comment:
the user still gets the option to disable it. Previously we had/have a way only turn on the breadcrumbs setting after 10 joined rooms.

Also the faster way to do this is `this.matrixClient.getVisibleRooms().length > 20`